presto-docs/src/main/sphinx/connector/inmemory.rst

```diff
@@ -0,0 +1,55 @@
+==================
+InMemory Connector
+==================
+
+The InMemory connector stores all data and metadata in RAM on workers
+and both are discarded when Presto restarts.
+
+.. warning::
+
+    This connector is in early experimental stage it is not recommended
```

Comment:
Please add something like: "The intended use of this connector at this point is purely for testing"